### PR TITLE
Revert to base 8.3 rpm-ostree

### PR DESCRIPTION
**Pull request by cgwalters**
We need to tag more packages to make this work.

### CODE REVIEW

File: rhcos-packages.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ packages:
   # needed by rpm-ostree for managing users
   - nss-altfiles
   - ostree
-  - "'rpm-ostree >= 2020.7'"
+  - rpm-ostree-2020.7-1.el8
```
Comment (travier) [inline rhcos-packages.yaml:18]: Don't we need `2020.4` so `<2020.7`?

Comment (cgwalters) [reply rhcos-packages.yaml]: Yep done, thanks!

**Review by ashcrow — APPROVED** [review]

**Pull request merged**